app/controllers/group_memberships_controller.rb

```diff
@@ -0,0 +1,11 @@
+class GroupMembershipsController < AuthenticationController
+  def create
+    result = CreateGroupMembership.call(user_id:  current_user.id,
+                                        group_id: params[:group_id])
+    if result.success?
+      head :no_content
```

Comment:
That makes sense to me too. Usually you need to return something if you generated some new data that the client doesn't have. In this case the client already has everything needed.